frontend/packages/insights-plugin/src/components/InsightsPopup/style.scss

```diff
@@ -0,0 +1,5 @@
+.co-insights__box {
+	.co-insights__no-rules {
```

Comment:
We typically avoid nesting of CSS styles and find it's not often necessary with BEM. Is `.co-insights__no-rules` by itself specific enough here?